### ISSUE TITLE
New warning in 5.2.1

### ISSUE THREAD

**Issue by keith** [card-io/card.io-iOS-SDK#154]
After updating from 5.2.0 to 5.2.1 we started getting a new warning:

```
ld: warning: ignoring file ~/project/Pods/CardIO/CardIO/libopencv_core.a, missing required architecture x86_64 in file ~/project/Pods/CardIO/CardIO/libopencv_core.a (3 slices)
ld: warning: ignoring file ~/project/Pods/CardIO/CardIO/libopencv_imgproc.a, missing required architecture x86_64 in file ~/project/Pods/CardIO/CardIO/libopencv_imgproc.a (3 slices)
```

It looks like the opencv static libraries weren't compiled as they should have been. You can see the difference in architectures vs the CardIO static library:

```sh
% file libopencv_core.a
libopencv_core.a: Mach-O universal binary with 3 architectures
libopencv_core.a (for architecture armv7s):     current ar archive random library
libopencv_core.a (for architecture armv7):      current ar archive random library
libopencv_core.a (for architecture arm64):      current ar archive random library
% file libCardIO.a
libCardIO.a: Mach-O universal binary with 5 architectures
libCardIO.a (for architecture armv7):   current ar archive random library
libCardIO.a (for architecture armv7s):  current ar archive random library
libCardIO.a (for architecture i386):    current ar archive random library
libCardIO.a (for architecture x86_64):  current ar archive random library
libCardIO.a (for architecture arm64):   current ar archive random library
```

**Comment by dgoldman-ebay** [comment]
Since the iOS simulator lacks a camera, the simulator builds (`x86_64`, `i386`) are not supposed to include the OpenCV libraries. All OpenCV calls in card.io are under an appropriate compiler flag.

Thus, the OpenCV static libraries do not need to be built for `x86_64` and `i386`.

Sounds like the project is failing to exclude these libraries for these architectures.

**Comment by keith** [comment]
Ah interesting, that makes sense. We're including CardIO through CocoaPods, so I assume anyone using CocoaPods for this would see the same issue.

**Comment by bluk** [comment]
I am labeling this as an enhancement only because this seems to be fairly harmless right now. For CocoaPods, http://blog.carbonfive.com/2014/01/24/cocoapods-for-device-only-ios-libraries/ seems to be a way to change the Podspec; there didn't seem to be an easier way in a quick search.

**Comment by bluk** [comment]
Should be fixed in release [5.2.2](https://github.com/card-io/card.io-iOS-SDK/releases/tag/5.2.2). The x86_64 and i386 slices were added. Even though they aren't used, removing them presented problems with Carthage support, so the short term solution is a temporary size increase in the OpenCV libraries.
